### ISSUE TITLE
auth_emailadmin - No uninstall

### ISSUE THREAD

**Issue by Lestices** [hrimhari/moodle-auth_emailadmin#44]
Hello, there is no uninstall option on the plugins overview.
Have you got a solution ?
Thks

**Comment by hrimhari** [comment]
No, there's no uninstall.As alternatives, you can remove the binaries or disable the plug-in.Sorry,FelipeSent from my Samsung Galaxy smartphone.
-------- Original message --------From: Lestices <notifications@github.com> Date: 2020-08-28  15:59  (GMT-05:00) To: hrimhari/moodle-auth_emailadmin <moodle-auth_emailadmin@noreply.github.com> Cc: Subscribed <subscribed@noreply.github.com> Subject: [hrimhari/moodle-auth_emailadmin] auth_emailadmin - No uninstall (#44) 
Hello, there is no uninstall option on the plugins overview.
Have you got a solution ?
Thks
